src/Evolution/Systems/ScalarWave/Tags.hpp

```diff
@@ -30,4 +30,24 @@ struct Phi : db::SimpleTag {
   using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
   static std::string name() noexcept { return "Phi"; }
 };
+
+namespace Tags {
+/*!
+ * \brief Tags corresponding to various constraints of the scalar
+ * wave system, and their diagnostically useful combinations.
+ * \details For details on how these are defined and computed, see
+ * `OneIndexConstraintCompute`, `TwoIndexConstraintCompute`
+ */
+template <size_t Dim>
+struct OneIndexConstraint : db::SimpleTag {
+  using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
+  static std::string name() noexcept { return "OneIndexConstraint"; }
```

Comment:
I thought this was true of option tags. 

If I remove the `name()` functions from these simple tags, the auto-generated names appear to **not** be inherited by the corresponding compute tags. As a result, my tests of compute-tag names, e.g. on `ScalarWave::Tags::OneIndexConstraintCompute<dim>::name()`, fail to compile.

Reply:
- DataBox tag names also default to the struct name now
- Better give the compute item an explicit name I think, but I don't think we have a policy for that yet.

Reply:
> * DataBox tag names also default to the struct name now
> * Better give the compute item an explicit name I think, but I don't think we have a policy for that yet.

I would think its better to have `name()` of a compute item come from the simple tag that it is inheriting from. I will go ahead and remove the name from these simple tags, and also remove their tests since those do not compile without the `name()` member.

Reply:
I think it will generate the compute tag names correctly if you add `using base = WhateverSimpleTag;` to their definitions.

Reply:
@prayush don’t remove the name tests, but just update them to use `db::tag_name<...>()`

Reply:
okay, I removed `name()`s in simple tags, kept the tests using `db::tag_name`